src/modules/account/components/AccountComponent.vue

```diff
@@ -66,7 +66,14 @@ setup(p){
       return isMulti;
     });  
     const accountName = ref(acc.name);
-    const accountNameDisplay = ref(acc.name);
+    const accountNameDisplay = computed(()=>{
```

Comment:
@anakin001 
`const accountNameDisplay = computed(()=>{
      return walletState.currentLoggedInWallet.convertAddressToName(p.address, true);
    });`

Reply:
Ok, will check on other modules and update accordingly